x-pack/plugins/security_solution/public/app/routes.tsx

```diff
@@ -14,6 +14,9 @@ import { HomePage } from './home';
 import { ManageRoutesSpy } from '../common/utils/route/manage_spy_routes';
 import { RouteCapture } from '../common/components/endpoint/route_capture';
 import { AppAction } from '../common/store/actions';
+import { useKibana } from '../common/lib/kibana';
+
+// import { ML_PAGES } from '../../../ml/public'; // not ok import
```

Comment:
Hard for other plugins to access the definitions or the constants from ml if they don't have our plugins as a dependency

Reply:
`// not ok import`
it is fine to import from other plugin's if it has been purposely shared.